### ISSUE TITLE
$HOME is /home instead of /root

### ISSUE THREAD

**Issue by mayureshw** [corbinlc/GNURootDebian#47]
Just installed the latest GNURootDebian on one device.

Noticed that $HOME points to /home instead of /root

Did I miss installing anything?

**Comment by corbinlc** [comment]
That is correct.  I will make it so the $HOME environment variable behaves normally in the near future.

**Comment by AdamMatthewLTM** [comment]
Hi all,

I am currently assisting the developers in sorting relevant issues for GNURoot Debian on GitHub. If the issue discussed is still relevant, could you please reply to this thread and state the issue needs to be resolved and I will attempt to sort it for you ASAP. If the issue is no longer relevant, please reply to this thread and state it has been resolved, and I will suggest this thread be closed.

Furthermore, if a particular issue is considered to be high priority, please state that you are requesting it to be considered as high priority and I will mark it as such.

Thank you all so much for your time and understanding,
-Adam

**Comment by corbinlc** [comment]
We are putting our efforts into a new, but similar, app called UserLAnd.  It is now in beta and it has the HOME variable set the way it normally is on linux systems.  To join the beta of UserLAnd, please go here:  https://play.google.com/apps/testing/tech.ula
